### DISCUSSION TITLE
Is the API to set mss deliberately missing?

### DISCUSSION THREAD

**Discussion by WatcherOfTheSkies** [esnet/iperf#1279] [Q&A] [answered]
Hi,

Please can you tell me whether the API to set mss is deliberately missing? 
Along the lines of...

void
iperf_set_test_mss(struct iperf_test *ipt, int mss)
{
    ipt->settings->mss = mss;
}

Thank you!

**Comment by bmah888** [comment, marked as answer]
Short answer: It wasn't deliberately left out as far as I can remember. Just a silly omission. We'd take a pull request for it, from anyone so inclined.
